src/components/Button/Button.tsx

```diff
@@ -18,18 +19,16 @@ export const Button = ({
     setTimeout(() => {
       setIsLoading(false);
     }, 1000);
```

Comment:
can we avoid hardcoded timeouts? its better to place setIsLoading(true); somewhere in top of the async function and setIsLoading(false) on the bottom